### DISCUSSION TITLE
Livewire in screen

### DISCUSSION THREAD

**Discussion by hasanmonfared** [orchidsoftware/platform#1977] [Q&A]
Hi, I use the orchid platform. For some parts, I want to use livewire. Everything is fine, but when I want to use the modal bootstrap and when we submit, I have this interference with the js file orchid in livewire, and I have this error and a toast is displayed.
Thank you if you have a solution for this issue.

```
An invalid form control with name='' is not focusable.
value @ orchid.js?id=25a1be8cce5b3896cd46:16
value @ orchid.js?id=25a1be8cce5b3896cd46:16
invokeWithEvent @ orchid.js?id=25a1be8cce5b3896cd46:2
handleEvent @ orchid.js?id=25a1be8cce5b3896cd46:2
handleEvent @ orchid.js?id=25a1be8cce5b3896cd46:2
```

**Comment by hoangnamitc** [comment]
This package support livewire ?
